manifests/contact.pp

```diff
@@ -0,0 +1,55 @@
+# = Define: sensu::routing
+#
+# Manage [Contact
+# Routing](https://sensuapp.org/docs/0.26/enterprise/contact-routing.html)
```

Comment:
we're on 0.29. Best to use latest here though

`https://sensuapp.org/docs/latest/enterprise/contact-routing.html`